tests/baselines/reference/superInConstructorParam1.js

```diff
@@ -27,7 +27,7 @@ var B = (function () {
 var C = (function (_super) {
     __extends(C, _super);
     function C(a) {
-        if (a === void 0) { a = _super.prototype.foo.call(this); }
+        if (a === void 0) { a = _super.foo.call(this); }

```

Comment:
do not think it matters much as this access is invalid before super, so in ES6 that would be a runtime error, i presume